user-contrib/Ltac2/tac2entries.ml

```diff
@@ -322,7 +325,7 @@ let check_lowercase {loc;v=id} =
   if Tac2env.is_constructor (Libnames.qualid_of_ident id) then
     user_err ?loc (str "The identifier " ++ Id.print id ++ str " must be lowercase")
 
-let register_ltac ?(local = false) ?(mut = false) isrec tactics =
+let register_ltac ?deprecation ?(local = false) ?(mut = false) isrec tactics =
```

Comment:
Why not default to "not deprecated"?

Reply:
That's the default already.

Reply:
No, the `?` syntax means that if you don't pass an argument you'll get None, which means not deprecated.

Reply:
Oh